Dockerfile

```diff
@@ -35,58 +35,26 @@ RUN cat /usr/local/bin/start.sh
 
 USER $NB_USER
 
+RUN mkdir /home/$NB_USER/tmp
+COPY base-env.yml /home/$NB_USER/tmp/base-env.yml
 # Install Python 3 packages
 # Remove pyqt and qt pulled in for matplotlib since we're only ever going to
 # use notebook-friendly backends in these images
 
-RUN conda install -c bokeh --yes \
-    datashader
+RUN conda env update --name root --file /home/$NB_USER/tmp/base-env.yml 
```

Comment:
@lsetiawan @amanda-tan due to https://github.com/conda/conda/issues/7626 we would need to use only `defaults` or `conda-forge`, but not both. B/c there are some `conda-forge` specific packages in the environments I guess we need to remove `defaults`.

To start we need to run:

```shell
conda config --remove channels defaults
```

above this line.